server/cmd/mmctl/commands/post.go

```diff
@@ -100,13 +106,34 @@ func postCreateCmdF(c client.Client, cmd *cobra.Command, args []string) error {
 		}
 	}
 
-	channel := getChannelFromChannelArg(c, args[0])
-	if channel == nil {
-		return errors.New("Unable to find channel '" + args[0] + "'")
+	channelID := ""
+	if strings.HasPrefix(args[0], "@") {
+		me, _, err := c.GetMe(context.TODO(), "")
+		if err != nil {
+			return fmt.Errorf("could not get current user: %s", err.Error())
+		}
+
+		userArgString := strings.Split(args[0], "@")[1]
```

Comment:
Do we need to check if `[1]` exists?

Reply:
It's needed to prevent `index out of range` when user fails to passs the second arg